### PR TITLE
Improves WP_REST_Posts_Controller documentation.

### PR DESCRIPTION
**Pull request by Soean**

### CODE REVIEW

**Comment by rachelbaker** [comment]
@Soean thanks you for the patch.  WP Core's documentation standard only uses the `@access` property for functions.  See https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods

**Comment by Soean** [comment]
@rachelbaker thanks for the feedback. Core uses `@access` also for properties. See https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#2-1-class-members
example: https://core.trac.wordpress.org/browser/tags/4.6/src/wp-includes/rest-api/class-wp-rest-response.php

**Comment by rachelbaker** [comment]
@Soean That is correct.  So keep the `@access` property on the properties, but remove from the methods and this will be 👍

**Comment by Soean** [comment]
@rachelbaker Here is my refresh

**Comment by kadamwhite** [comment]
@Soean +1 on the change; if you can fix the merge conflict (merge latest `develop` into this branch) this is ready to go!

File: lib/endpoints/class-wp-rest-posts-controller.php [hunk 1]
```diff
@@ -677,8 +688,8 @@ public function delete_item( $request ) {
 	 * Determine the allowed query_vars for a get_items() response and
 	 * prepare for WP_Query.
 	 *
-	 * @param array           $prepared_args
-	 * @param WP_REST_Request $request
+	 * @param array           $prepared_args Prepared WP_Query Arguments.
```
Comment (rachelbaker) [inline lib/endpoints/class-wp-rest-posts-controller.php:691]: `Arguments` should be lower case here.

**Review by rachelbaker — APPROVED** [review]

**Pull request merged**